dali/python/nvidia/dali/pipeline.py

```diff
@@ -92,11 +92,15 @@ class Pipeline(object):
     unrestricted number of streams is assumed).
 `default_cuda_stream_priority` : int, optional, default = 0
     CUDA stream priority used by DALI. See `cudaStreamCreateWithPriority` in CUDA documentation
+`get_memory_stats`: bool, optional, default = False
+    If DALI should print operator output buffer statistics.
+    Usefull for `bytes_per_sample_hint` operator parameter.
 """
     def __init__(self, batch_size = -1, num_threads = -1, device_id = -1, seed = -1,
                  exec_pipelined=True, prefetch_queue_depth=2,
                  exec_async=True, bytes_per_sample=0,
-                 set_affinity=False, max_streams=-1, default_cuda_stream_priority = 0):
+                 set_affinity=False, max_streams=-1, default_cuda_stream_priority = 0,
+                 get_memory_stats=False):
```

Comment:
As we're adding new arguments and we're using Python 3 only, you can maybe place a `*` before it, so it's only allowed to be a Keyword argument.
I don't think it's good to encourage usage like `Pipeline(2, 4, 1, 0, True, 3, True, 0, True, 4, 3, False)`

Reply:
Done